integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -23,7 +23,7 @@ export function importAppBefore(
   logLevel: Realm.App.Sync.LogLevel = (environment.syncLogLevel as Realm.App.Sync.LogLevel) || "warn",
 ): void {
   before(async function (this: Partial<AppContext> & Mocha.Context) {
-    this.timeout(10000);
+    this.timeout(120000); // Let's give it 2 minutes
```

Comment:
Wow, the import can take 2 minutes?

Reply:
Yeah - I was surprised. I notice this is fast when there's no database connection, I think it might be adding a change to the MongoDB server deployment to allow access from BaaS. Applying an atlas configuration takes ~ 1 minute from my experience.

Reply:
Ah OK, might be worth changing the comment to give a bit more context? We could change the timeout to `2 * 60 * 1000` to make the two seconds part clear